src/Xamarin.Android.Build.Tasks/Tasks/GenerateJavaStubs.cs

```diff
@@ -126,21 +126,28 @@ void Run (DirectoryAssemblyResolver res)
 			bool haveMonoAndroid = false;
 			var allTypemapAssemblies = new HashSet<string> (StringComparer.OrdinalIgnoreCase);
 			var userAssemblies = new Dictionary<string, string> (StringComparer.OrdinalIgnoreCase);
+			var tokenIdCollection = new NativeRuntimeTokenIdCollection (Log);
 			foreach (var assembly in ResolvedAssemblies) {
 				bool value;
+				string fileName = Path.GetFileName (assembly.ItemSpec);
+
+				if (String.Compare ("Java.Interop.dll", fileName, StringComparison.CurrentCultureIgnoreCase) == 0) {
+				    tokenIdCollection.ProcessJavaInterop (assembly.ItemSpec);
+				}
+
 				if (bool.TryParse (assembly.GetMetadata (AndroidSkipJavaStubGeneration), out value) && value) {
```

Comment:
(The joys of time and broader context…)

@dellis1972 : should commit 21561c3b11a0b7763e8bf2ed9f718e6a8f2446ce / PR #2899 have *also* set `%(AndroidSkipJavaStubGeneration)` on `Java.Interop.dll` and `Mono.Android.Export.dll`?  (And, *at the time*, `OpenTK-1.0.dll`?  Though that's needed *now* as of f909adb146c0ee87dafad2417e301a14afb79a33 …)

Reply:
They probably should have been included at that time yes.